src/controllers/announcement_controller.cr

```diff
@@ -114,4 +118,17 @@ class AnnouncementController < ApplicationController
   private def user_id_param
     User.find_by_login(params["user"]?).try(&.id)
   end
+
+  private def check_rate_limit!
+    return true unless rate_limit = max_anns_per_hour
+    if current_user!.last_hour_announcements >= rate_limit
+      return redirect_to "/"
+    end
+  end
+
+  private def limit_reached?
+    if (rate_limit = max_anns_per_hour) && signed_in?
+      return current_user!.last_hour_announcements + 1 > rate_limit
```

Comment:
`return` is unnecessary in both methods, but it seems to be a matter of style.

Reply:
This ```return``` is a leftover from previous implementation of this method, we can remove it!